src/theme/components/customTable.ts

```diff
@@ -35,11 +44,13 @@ const baseStyle = ({
   },
   caption: {
     captionSide: "top",
-    textAlign: "left",
-    fontSize: "heading.secondary",
     color: "ui.black",
-    textTransform: "capitalize",
+    fontSize: "heading.secondary",
     fontWeight: "heading.secondary",
+    margin: "0",
```

Comment:
Do you think that CSS issue would be resolved if each margin direction was set separately?

Reply:
Doesn't hurt to do.